kadena-signing-api/src/Kadena/SigningTypes.hs

```diff
@@ -32,21 +33,21 @@ newtype SignatureList =
   deriving (Eq, Ord, Show, Semigroup, Monoid, Generic)
 
 instance ToJSON SignatureList where
-  toJSON (SignatureList sl ) = object $
-    map (\(k,ms) -> (unPublicKeyHex k, maybe Null (toJSON . _usSig) ms)) sl
+  toJSON (SignatureList rawLst) = Array $ V.fromList $
```

Comment:
I like this change because it makes it easier for tooling to sign transactions without needing to actually parse the underlying command to generate a finalized transaction.